### PR TITLE
[SYNTH-11931] Create README for the integration

### PR DESCRIPTION
**Pull request by teodor2312**
Fixing the README file for this integration to show relevant information

### CODE REVIEW

**Comment by teodor2312** [comment]
For reference this PR is very similar to this one 
https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/pull/8
so most comments were left on that one but applied to both

File: README.md [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
-placeholder description
+<!-- TODO add link to marketplace after we publish the step -->
+<!-- [![Visual Studio Marketplace Version]()][1001]  -->
+[![Build Status](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4/status.svg?token=480MdFpG78E6kZASg5w1dw&branch=main)](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# Overview
 
-## How to use this Step
+With the `synthetics-test-automation-bitrise-step-upload-application` step you can upload a new version of your application to Datadog to run Synthetics tests against during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2002]. Your application already needs to exist for this to work.
```
Comment (jhgilbert) [inline README.md:10]:
```suggestion
With the `synthetics-test-automation-bitrise-step-upload-application` step, you can upload a new version of your application to Datadog to run Synthetics tests against during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle. This step uses the [Datadog CI Synthetics command][2002]. Your application already needs to exist for this to work.
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,35 @@
...
-and call `bitrise run test`.
+## Setup
 
-*Check the `bitrise.yml` file for required inputs which have to be
-added to your `.bitrise.secrets.yml` file!*
+To get started:
 
-Step by step:
+1. Add this step to your workflow. You can see [Bitrise's documentation][4001] on how to do that. You can also configure it locally by referencing this step in your `bitrise.yml`.
+2. Add your API and App Keys to your secrets in Bitrise. Documentation on how to do that can be found here - [Setting a Secret][4002].
```
Comment (jhgilbert) [inline README.md:17]:
```suggestion
2. Add your API and App keys to your secrets in Bitrise. Documentation on how to do that can be found here: [Setting a Secret][4002].
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,35 @@
...
+3. Configure your step inputs ([Step Inputs][4003]). You can also configure them in your `bitrise.yml`. The only required inputs are the 2 secrets you configured earlier. The rest of the possible inputs are described in a later section.
```
Comment (jhgilbert) [inline README.md:18]:
```suggestion
3. Configure your step inputs (see the [Step Inputs documentation][4003]). You can also configure them in your `bitrise.yml`. The only required inputs are the two secrets you configured earlier. The rest of the possible inputs are described in a later section.
```

File: README.md [hunk 4]
```diff
@@ -1,27 +1,35 @@
...
+
+## How to use this Step locally
```
Comment (jhgilbert) [inline README.md:20]:
```suggestion
## How to use this step locally
```

File: README.md [hunk 5]
```diff
@@ -1,27 +1,35 @@
...
+
+This Step can be run directly with the [Bitrise CLI][2003].
```
Comment (jhgilbert) [inline README.md:22]:
```suggestion
This step can be run directly with the [Bitrise CLI][2003].
```

File: README.md [hunk 6]
```diff
@@ -31,63 +39,78 @@ envs:
...
 
-**NOTE:**
+For an example configuration file, see the [`global.config.json` file][2001].
 
-If you want to use your step in your project's `bitrise.yml`:
+### Example including all possible configurations
 
-1. git push the step into it's repository
-2. reference it in your `bitrise.yml` with the `git::PUBLIC-GIT-CLONE-URL@BRANCH` step reference style:
+For reference here's how a full configuration could look like:
```
Comment (jhgilbert) [inline README.md:61]:
```suggestion
For reference here's how a full configuration could look:
```

**Review by jhgilbert — APPROVED** [review]
Approving with the same suggestions as in the other PR, just for convenience on your side. Thank you!

File: README.md (hunk 4 above)
Comment (teodor2312) [reply README.md:20]: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/pull/8#discussion_r1504082222

File: README.md [hunk 7]
```diff
@@ -1,27 +1,35 @@
-# datadog-mobile-app-upload
+# Datadog Continuous Testing for Bitrise
 
-placeholder description
+<!-- TODO add link to marketplace after we publish the step -->
+<!-- [![Visual Studio Marketplace Version]()][1001]  -->
+[![Build Status](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4/status.svg?token=480MdFpG78E6kZASg5w1dw&branch=main)](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4)
```
Comment (Drarig29) [inline README.md:5]: I noticed the link to the build status was wrong in the other PR, but correct here 🤔 

https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/pull/12

**Review by Drarig29 — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (teodor2312) [reply README.md]: It was correct but actually pointing to the wrong place 😅

**Pull request merged**